samples/apps/copilot-chat-app/SKWebApi/appsettings.json

```diff
@@ -1,34 +1,33 @@
 {
+  // Consider populating secrets, such as "Key" properties, from dotnet's user secrets when running locally.
+  // https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-7.0&tabs=windows#secret-manager
+  // Ex: dotnet user-secrets set "CompletionConfig:Key" "MY_SECRET_VALUE"
+  // Values in user secrets take precedence over those in this file
+
   "ServicePort": "40443",
   "CompletionConfig": {
     "Label": "Completion",
-    "AIService": "", // "AzureOpenAI" or "OpenAI"
-    "DeploymentOrModelId": "text-davinci-003",
-    "Endpoint": "", // Ex: "https://YOUR-INSTANCE.openai.azure.com/"
-    // Consider populating the value below from user secrets when running locally.
-    // Ex: dotnet user-secrets set "CompletionConfig:Key" "MY_SECRET_VALUE"
-    // Values in user secrets take precedence over those in this file
-    "Key": "**DO NOT CHECK IN**"
+    "AIService": "[Enter 'AzureOpenAI', or 'OpenAI']",
+    "DeploymentOrModelId": "[Enter the Azure OpenAI deployment name or OpenAI model, e.g. 'text-davinci-003']",
+    "Endpoint": "[Enter the endpoint of your Azure OpenAI instance, e.g. https://contoso.openai.azure.com]",
+    "Key": "[DO NOT CHECK IN - Enter the key to your Azure OpenAI instance.]"
   },
   // Remove or comment out EmbeddingConfig if no embedding backend is required
   "EmbeddingConfig": {
     "Label": "Embeddings",
-    "AIService": "", // "AzureOpenAI" or "OpenAI"
-    "DeploymentOrModelId": "text-embedding-ada-002",
-    "Endpoint": "", // Ex: "https://YOUR-INSTANCE.openai.azure.com/"
-    // Consider populating the value below from user secrets when running locally.
-    // Ex: dotnet user-secrets set "EmbeddingConfig:Key" "MY_SECRET_VALUE"
-    // Values in user secrets take precedence over those in this file
-    "Key": "**DO NOT CHECK IN**"
+    "AIService": "[Enter 'AzureOpenAI', or 'OpenAI']",
+    "DeploymentOrModelId": "[Enter the Azure OpenAI deployment name or OpenAI model, e.g. 'text-embedding-ada-002']",
+    "Endpoint": "[Enter the endpoint of your Azure OpenAI instance, e.g. https://contoso.openai.azure.com]",
+    "Key": "[DO NOT CHECK IN - Enter the key to your Azure OpenAI instance.]"
```

Comment:
if we remove the comments alerting people let's also remove the property to avoid the risks. We had too many incidents with secrets ending in the repo

Reply:
as with the previous comment, we will cover this in a follow-up PR.